build/target-repository/docs/rector_rules_overview.md

```diff
@@ -8186,6 +8185,33 @@ Add return method return type based on strict typed property
 
 <br>
 
+### ReturnUnionTypeRector
+
+Add union return type
+
+- class: [`Rector\TypeDeclaration\Rector\ClassMethod\ReturnUnionTypeRector`](../rules/TypeDeclaration/Rector/ClassMethod/ReturnUnionTypeRector.php)
+
+```diff
+ final class SomeClass
+ {
+-    public function getData()
++    public function getData(): null|\DateTime|\stdClass
```

Comment:
I think it makes sense. I try to prevent methods returning unions with more then 2 types, but for legacy projects its still a great thing to get type native type coverage without prior refactoring. 


So 👍